tracer/src/Datadog.Trace/Propagators/SpanContextPropagator.cs

```diff
@@ -55,7 +57,8 @@ public static SpanContextPropagator Instance
                         {
                             DistributedContextExtractor.Instance,
                             DatadogContextPropagator.Instance
-                        });
+                        },
+                        false);
```

Comment:
I guess this is a workaround for the issue we discussed before? 😄 Do we know what the implications of this are? When will we hit this code path? 

Off the top of my head, I'm guessing it's only if (for example) 

- A customer is using manual-only instrumentation
- They call `SpanContextPropagator.Instance` _before_ calling `Tracer.Configure()`

If that's the only case, this seems like a much better solution than what I suggested before! 🙈

Reply:
Yeah before I had errors because I added the changes incorrectly so it wasn't working for the tests, now it works good!
About the implications I came to the conclusion that if any implications it would be for the minority of customers doing this setups that we are trying to walk away from so I imagine it should be fine.